### PR TITLE
Change type of screenshots from input to textarea

### PR DESCRIPTION
**Pull request by fireddd**
This PR is addressing the following issue #2027
https://github.com/EddieHubCommunity/support/issues/2027

### CODE REVIEW

**Review by fireddd — COMMENTED** [review]
Changing the type of the screenshots

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for your contribution

File: .github/ISSUE_TEMPLATE/ideas.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ body:
       label: Description of feature request, idea
     validations:
       required: true
-  - type: input
+  - type: textarea
```
Comment (Akshu-on-github) [inline .github/ISSUE_TEMPLATE/ideas.yml:12]: LGTM ✅

**Review by Akshu-on-github — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for the fix! 😄👍
I would suggest updating the other files for this too. If you want, you can implement it in this same PR or let others do it. 🙂

**Review by krishdevdb — APPROVED** [review]

**Review by fireddd — COMMENTED** [review]
Updated all files having screenshot field

**Review by Vyvy-vi — APPROVED** [review]
LGTM! (however, in future maybe it might make sense to convert the `bug` and `ideas` labels back to their normal versions)

**Comment by Vyvy-vi** [comment]
Thanks for the contribution! :tada:

**Pull request merged**